.eslintrc.yml

```diff
@@ -40,10 +40,10 @@ rules:
 
   #flowtype/array-style-complex-type: undecided
   #flowtype/array-style-simple-type: undecided
-  #flowtype/boolean-style: undecided
+  flowtype/boolean-style: off
```

Comment:
Based on this list:
https://github.com/prettier/eslint-config-prettier/blob/master/flowtype.js